client/app/pages/alert/Alert.jsx

```diff
@@ -113,12 +117,12 @@ class Alert extends React.Component {
 
     return AlertService.save(alert)
       .then(alert => {
-        notification.success("Saved.");
+        notification.success(i18next.t("Saved."));
```

Comment:
Should we have the `Alerts:` prefix here too?

```suggestion
        notification.success(i18next.t("Alerts:Saved."));
```

Reply:
because "Saved." is used in many places across the app, i did put it under the `common` namespace which is set as default, so we don't need to write `common:`